backend/tracim_backend/models/context_models.py

```diff
@@ -681,6 +694,20 @@ def agenda_enabled(self) -> bool:
         """
         return self.workspace.agenda_enabled
 
+    @property
+    def public_download_enabled(self) -> bool:
+        """
+        returns True if public download is enabled in this workspace
+        """
+        return self.workspace.public_download_enabled
+
+    @property
+    def public_upload_enabled(self) -> bool:
+        """
+        returns True if public upload is enabled in this workspace
+        """
+        return self.workspace.public_upload_enabled
+
```

Comment:
Where is used these two perperties ? Are they really required ?

Reply:
(If not useful, remove it. After that this merge request is mergeable)

Reply:
These are workspace properties, this allow to allow/disallow public download (content share) or public upload (upload permission) on specific workspace.

Reply:
Okay, i missready github diff render :p